compiler/src/dotty/tools/dotc/transform/CheckJVMSpecification.scala

```diff
@@ -0,0 +1,43 @@
+package dotty.tools.dotc
+package transform
+
+import MegaPhase.MiniPhase
+import GenericSignatures.*
+import core.Contexts.*
+import core.Decorators.*
+import core.Constants.*
+import core.Symbols.*
+import util.SrcPos
+
+final class CheckJVMSpecification extends MiniPhase:
```

Comment:
Consider an `end CheckJVMSpecification` marker for this class. It spans more than one screen.